app.py

```diff
@@ -13,10 +13,25 @@
 # Create App and stacks
 app = core.App()
 
+artifact_bucket = app.node.try_get_context("ArtifactBucket")
```

Comment:
Can you add the "drift:" prefix as per the other `cdk.json` values.

Reply:
Done